cfgov/regulations3k/wagtail_hooks.py

```diff
@@ -45,6 +48,34 @@ class EffectiveVersionModelAdmin(TreeModelAdmin):
     child_model_admin = SubpartModelAdmin
     parent_field = 'part'
 
+    def copy(self, instance):
+        subparts = instance.subparts.all()
+
+        # Copy the instance
+        new_version = instance
+        new_version.pk = None
+        new_version.acquired = date.today()
```

Comment:
Depending on whether this or #4387 gets merged first, this will have to be changed to the "created" field.